lib/internal/modules/esm/fetch_module.js

```diff
@@ -44,44 +44,67 @@ const cacheForGET = new SafeMap();
 // [2] Creating a new agent instead of using the gloabl agent improves
 // performance and precludes the agent becoming tainted.
 
+/** @type {import('https').Agent} The Cached HTTP Agent for **secure** HTTP requests. */
 let HTTPSAgent;
-function HTTPSGet(url, opts) {
+/**
+ * Make a **secure** HTTP GET request (handling agent setup if needed, caching the agent to avoid
```

Comment:
What's secure about it?
```suggestion
 * Make a HTTPs GET request (handling agent setup if needed, caching the agent to avoid
```